docs/src/main/asciidoc/_admin-guide/subsystem-configuration/EE_Concurrency_Utilities_Configuration.adoc

```diff
@@ -1,8 +1,8 @@
 [[EE_Concurrency_Utilities_Configuration]]
 = EE Concurrency Utilities
 
-EE Concurrency Utilities (JSR 236) were introduced with Java EE 7, to
```

Comment:
This should either remain as Java EE 7 or perhaps change to "EE Concurrency Utilities were introduced to ease the task of writing multithreaded Jakarta EE applications."  @emmartins what do you prefer?

Reply:
+1 to just dropping "Java EE 7".

Tangent: IMHO in general this kind of "introduced with XXX" text sprinkled through docs is more maintenance trouble than it is worth. Typically XXX is "WildFly <some_number> but we see here that even Java EE X adds some cost.

Reply:
Updating to:

> EE Concurrency Utilities (JSR 236) were introduced to ease the task of writing multithreaded applications. Instances of these utilities are managed by WildFly, and the related configuration.